route53_crossfeed_app.tf

```
@@ -2,36 +2,31 @@
 # Resource records that support Crossfeed site cloudfront endpoints and application.
 # ------------------------------------------------------------------------------
 
+###############################################################################
+# Look up the existing hosted zones for both cyber.dhs.gov and cisa.dhs.gov
+###############################################################################
+data "aws_route53_zone" "cyber_dhs_gov" {
+  provider     = aws.route53resourcechange
+  name         = "cyber.dhs.gov"
+  private_zone = false
+}
+
 # ------------------------------------------------------------------------------
 # Prod entries
 # ------------------------------------------------------------------------------
 
-# The hosted_zone_id for the below records comes from https://docs.aws.amazon.com/general/latest/gr/elb.html
-# (ALBs in us-gov-west-1 region)
-resource "aws_route53_record" "crossfeed_prod_A" {
+# CNAME so crossfeed.cyber.dhs.gov → cyber‑hygiene.cisa.dhs.gov
+resource "aws_route53_record" "crossfeed_prod_CNAME" {
   provider = aws.route53resourcechange
 
-  alias {
-    name                   = "crossfeed-prod-1638162291.us-gov-west-1.elb.amazonaws.com."
-    evaluate_target_health = false
-    zone_id                = "Z33AYJ8TM3BH4J"
-  }
-  name    = "crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
-  type    = "A"
-  zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
-}
-
-resource "aws_route53_record" "crossfeed_prod_AAAA" {
-  provider = aws.route53resourcechange
+  name    = "crossfeed.${data.aws_route53_zone.cyber_dhs_gov.name}"
+  ttl     = 300
+  type    = "CNAME"
+  zone_id = data.aws_route53_zone.cyber_dhs_gov.zone_id
```

This how to reference the `cyber.dhs.gov` zone ID:
```suggestion
  zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
```

---

Removed per Commit suggestion